OWNERS_ALIASES

```diff
@@ -4,73 +4,66 @@ aliases:
     - justaugustus
     - tpepper
   licensing:
-    - dims
-    - justaugustus
-    - nikhita
-    - swinslow
+    - dims # subproject owner
+    - justaugustus # subproject owner
+    - nikhita # subproject owner
+    - swinslow # subproject owner
   release-engineering:
-    - aleksandra-malinowska
-    - calebamiles
-    - ixdy
-    - mikedanese
-    - tpepper
+    - aleksandra-malinowska # Patch Release Team
+    - calebamiles # subproject owner
+    - dougm # Patch Release Team
+    - feiskyer # Patch Release Team
+    - hoegaarden # Patch Release Team
+    - idealhack # Patch Release Team
+    - justaugustus # subproject owner
+    - tpepper # subproject owner / Patch Release Team
```

Comment:
I it somehow possible for me to join release engineering, too? My plan was to continue contributions to the release notes processes from that perspective :innocent:

Reply:
@saschagrunert -- I'd be happy to have you working on the Release Engineering side, and definitely appreciate the work you do on release notes today.

To clarify this owner alias, it's the set of contributors who have wide encompassing knowledge of all code that lives within the repo. Currently, that's the Release Engineering subproject owners, which include the SIG Release Chairs and the Patch Release Team.

Folx interested in participating I would expect to become a Release Manager Associate, move into Branch Manager, and then into the Patch Release Team.

We'll look to firmly define the contributor ladder for Release Engineering in 1.17! :)

Reply:
Ah okay good to know thanks for the clarification. :)

Reply:
@saschagrunert -- here's the tracking issue for that: https://github.com/kubernetes/sig-release/issues/695